p4-16/psa/PSA.mdk

```diff
@@ -951,9 +948,9 @@ packet contents and metadata when a packet begins egress processing.
 +--------------+-------------+---------------+--------------+-----+
 | `packet_path`    | Same value as received by EgressParser above. ||||
 +--------------+-------------+---------------+--------------+-----+
-| `instance`         | 0 | from PRE           | 0 for cloned packets that are not multicast replicated. ||
-|                    |   | configuration      | value from PRE configuration of multicast group  ||
-|                    |   | of multicast group | for cloned packets that are multicast replicated ||
+| `instance`         | 0 | from PRE           | from PRE configuration  ||
+|                    |   | configuration      | of clone session        ||
+|                    |   | of multicast group |                         ||
```

Comment:
We say "from PRE configuration for multicast group" in this table as a shorter way to explain where the value of instance comes from, with a longer description in the later text.  Suggest doing the same here for cloned packets.  The longer description is later in the text.